prep-javascript-variables/index.html

```diff
@@ -0,0 +1,21 @@
+<!-- eslint-disable no-unused-vars -->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JavaScript Variables</title>
+  </head>
+  <body>
+    <script>
+      console.log(typeof 42);
+      console.log(typeof 'Hello');
+
+      const greeting = 'Hello, world!';
+      console.log(greeting);
+
+      const val = null;
+      console.log(val); // Logs: null
```

Comment:
Make sure you reread the instructions correctly! You should have a total of 5 different variables.

Each variable should have 2 `console.log`s. For example:

```
const fullName = 'Shawn Kost';
console.log('value of fullName:', fullName);
console.log('type of fullName:', typeof fullName)
```